config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -767,7 +767,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.7
+      base_ref: release-1.8
```

Comment:
In-progress test after this change:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-azure-master-vmss-capz/1634252933753212928

Reminder to check for results!